tck/src/main/java/org/reactivestreams/tck/SubscriberBlackboxVerification.java

```diff
@@ -287,7 +287,16 @@ public void required_spec209_blackbox_mustBePreparedToReceiveAnOnCompleteSignalW
       public void run(BlackboxTestStage stage) throws Throwable {
         final Publisher<T> pub = new Publisher<T>() {
           @Override
-          public void subscribe(Subscriber<? super T> s) {
+          public void subscribe(final Subscriber<? super T> s) {
+            s.onSubscribe(new Subscription() {
```

Comment:
Absolutely, I guess what we could do is to either put in a small comment explaining why, or we could throw AssertionErrors when unsupported invocations occur, or provide a more complete implementation—if we already have any in the TCK that is.